src/main/java/com/github/casperjs/casperjsrunner/CasperJSRunnerMojo.java

```diff
@@ -307,11 +317,34 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         final Collection<String> scripts = findScripts();
         final Result globalResult = executeScripts(scripts);
         getLogger().info(globalResult.print());
+        writeFailedTestCount(globalResult);
         if (!ignoreTestFailures && globalResult.getFailures() > 0) {
             throw new MojoFailureException("There are " + globalResult.getFailures() + " tests failures");
         }
     }
 
+    private void writeFailedTestCount(Result globalResult) throws MojoExecutionException {
+        try {
+            tryToWriteFailedTestCount(globalResult);
+        } catch (IOException e) {
+            throw new MojoExecutionException("", e);
+        }
+    }
+
+    private void tryToWriteFailedTestCount(Result globalResult) throws IOException {
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(testFailureCountFile, false));
```

Comment:
you should take care of creating the parent directories of the destination file it they don't exist, I think this is what make CI fails